contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,98 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the code of conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** In some cases we may determine that
+a public statement will need to be made. If that's the case, the identities of
+all victims, reporters, and the accused will remain confidential unless those
+individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
+  * If there is a publicly available record (e.g. a mailing list archive or a
+    public IRC or Slack log), please include a link.
+  * Any extra context you believe existed for the incident.
+  * If you believe this incident is ongoing.
+  * Any other information you believe we should have.
+
+What happens after you file a report?
+-------------------------------------
+
+You will receive a reply from the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+acknowledging receipt as soon as possible, but within 24 hours.
+
+The team will immediately meet to review the incident and determine:
```

Comment:
This means, that the incident will (has to?) be reviewed by more than one person?

Reply:
Maybe we need to clarify this topic .. basically in general yes all enforcement team members will review each case .. unless someone has recused themselves .. or the reporter specifically asks for an enforcement team member to not be included and the enforcement team agrees.

Reply:
I had the same question while reading the doc. You contact one person but the whole team has access to the case.